packages/terra-embedded-component/src/EmbeddedComponent.jsx

```diff
@@ -1,31 +1,144 @@
+/* global SystemJS */
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
-import styles from './EmbeddedComponent.scss';
+import 'systemjs/dist/system';
+import { loadStylesheet } from './assets';
 
-const cx = classNames.bind(styles);
+// Expose AMD required globals on the window
+window.define = window.SystemJS.amdDefine;
```

Comment:
Should we check the value of define before we override it?

Also, could we use an expose loader for this instead potentially? https://github.com/webpack-contrib/expose-loader